docs/how-to/installation/packages.txt

```diff
@@ -69,14 +155,25 @@ Configuring and Starting the Cluster
    Similarly, the agent can be run with the command
    ``determined-agent``.
 
+#. Verify that each agent started successfully by viewing the log:
+
+   .. code::
+
+      journalctl -u determined-agent
+
+   You should see logging indicating that the agent started successfully and
+   detected compute devices. On the Determined WebUI, you should now see slots
```

Comment:
Perhaps "started successfully, detected compute devices, and connected to the master".

Reply:
Done